gspread/worksheet.py

```diff
@@ -2607,6 +2607,25 @@ def unmerge_cells(self, name):
 
         return self.spreadsheet.batch_update(body)
 
+    def get_notes(self):
+        """Returns a list of lists containing all notes in the sheet, or the empty list if the
+        sheet does not have a note.
+        """
+        params = {"fields": "sheets/data/rowData/values/note"}
```

Comment:
I noticed this works, thought the official JSON field mask notation uses dot `.` to separate key, please replace all `/` with a `.` to be aligned with the official notation.